modules/modules.mk

```diff
@@ -62,19 +62,25 @@ endif
 
 ifeq ($(NAVIER_STOKES),yes)
         FLUID_PROPERTIES            := yes
-        HEAT_CONDUCTION             := yes
+        HEAT_TRANSFER               := yes
         RDG                         := yes
 endif
 
 ifeq ($(SOLID_PROPERTIES),yes)
-        HEAT_CONDUCTION             := yes
+        HEAT_TRANSFER               := yes
 endif
 
 ifeq ($(CONTACT),yes)
         TENSOR_MECHANICS            := yes
 endif
 
+# heat_conduction was renamed to heat_transfer
 ifeq ($(HEAT_CONDUCTION),yes)
```

Comment:
Is it possible for this to be yes considering your line removals above?

Reply:
if set externally

Reply:
also app makefiles still refer to heat_conduction in their makefile